src/plugins/console/public/lib/row_parser.ts

```diff
@@ -40,7 +40,7 @@ export default class RowParser {
       return MODE.IN_REQUEST;
     }
     let line = (this.editor.getLineValue(lineNumber) || '').trim();
-    if (!line || line[0] === '#') {
+    if (!line || line[0] === '#' || line.startsWith('//') || line.startsWith('/*')) {
```

Comment:
Can we be consistent in either using only `line[0]` or only `line.startsWith()`, but not both?